mne/io/meas_info.py

```diff
@@ -1211,3 +1214,141 @@ def _empty_info():
     info['nchan'] = info['sfreq'] = 0
     assert set(info.keys()) == set(RAW_INFO_FIELDS)
     return info
+
+
+class DigMontage(object):
+    """Montage for Digitized data
+
+    Montages are typically loaded from a file using read_montage. Only use this
+    class directly if you're constructing a new montage.
+
+    Parameters
+    ----------
+    hsp : array, shape (n_points, 3)
+        The positions of the channels in 3d.
```

Comment:
ok make explicit it is designed for Polhemus data. I would. Any chance we
want to support another system? if so I would put Polhemus in the function
name.